crates/driver/src/infra/api/routes/solve/dto/auction.rs

```diff
@@ -196,6 +203,7 @@ pub struct Auction {
     tokens: Vec<Token>,
     orders: Vec<Order>,
     deadline: chrono::DateTime<chrono::Utc>,
+    surplus_capturing_jit_order_owners: Option<Vec<eth::H160>>,
```

Comment:
The autopilot now sends this field unconditionally. Technically we only need `serde(Default)` to avoid errors in the shadow competition which receives requests from prod but uses the code from staging.
```suggestion
    #[serde(Default)]
    surplus_capturing_jit_order_owners: Vec<eth::H160>,
```